deploy/operator/deploy.sh

```diff
@@ -3,24 +3,64 @@
 set -o nounset
 set -o pipefail
 set -o errexit
-set -o xtrace
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(realpath ${__dir}/../..)"
 
 if [ -z "${DISKS:-}" ]; then
     export DISKS=$(echo sd{b..f})
 fi
 
+function handle_disconnected() {
+    # Some of the variables over here can be sourced from dev-scripts
+    # source common.sh utils.sh
+    # set +x
+    # export -f wrap_if_ipv6 ipversion
+
+    source ${__dir}/mirror_utils.sh
```

Comment:
import it top level, it shouldn't have any special effect to do this based on ``DISCONNECTED`` env